include/TrigScint/QIEInputPulse.h

```diff
@@ -129,6 +133,10 @@ class Expo : public QIEInputPulse {
    */
   Expo(float k, float tmax);  /// main constructor
 
+  /**
+   * Destructor
+   */
+  virtual ~Expo() = default;
```

Comment:
This is technically a change, but not something that will change any behavior. Still, noting it in the review